http/src/main/java/io/micronaut/http/hateos/JsonError.java

```diff
@@ -24,92 +24,28 @@
 import java.util.Optional;
 
 /**
- * A class that can be used to represent JSON errors that complies to Vnd.Error without the content type requirements.
+ * Deprecated. Please use io.micronaut.http.hateoas.JsonError
  *
  * @author Graeme Rocher
  * @since 1.0
  */
 @Produces(MediaType.APPLICATION_JSON)
-public class JsonError extends AbstractResource<JsonError> {
-
-    private String message;
-    private String logref;
-    private String path;
+@Deprecated
+public class JsonError extends io.micronaut.http.hateoas.JsonError {
 
     /**
      * @param message The message
      */
     public JsonError(String message) {
-        this.message = message;
+        super(message);
     }
 
     /**
      * Used by Jackson.
      */
     @Internal
+    //TODO AGB don't know how to solve this visibility problem
```

Comment:
I also don't know how to solve this visibility problem.
What do you think? Any ideas?